file.go

```diff
@@ -57,14 +55,11 @@ type File struct {
 	treatasclean bool // Window Clean tests should succeed if set. [private]
 
 	// Observer pattern: many Text instances can share a File.
-	curtext *Text
-	text    []*Text // [private I think]
+	*ObservableEditableBuffer //make this a pointer
```

Comment:
You have created a stand-alone PR here. And do not embed this.

Reply:
It's no longer embedded in the ObserverHasAFile as it's changed there